### ISSUE TITLE
Demo fails with a panic

### ISSUE THREAD

**Issue by 17twenty** [rdeusser/logsip#3]
```
$ go run logsip-demo.go 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x471dc6]

goroutine 1 [running]:
log.(*Logger).Output(0x0, 0x2, 0xc8200143c0, 0x28, 0x0, 0x0)
    /usr/local/go/src/log/log.go:147 +0x4e6
log.(*Logger).Print(0x0, 0xc820043dc0, 0x1, 0x1)
    /usr/local/go/src/log/log.go:178 +0x6a
github.com/iamthemuffinman/logsip.(*Logger).Info(0xc82001a120, 0xc820043f10, 0x1, 0x1)
    /home/nick/go/src/github.com/iamthemuffinman/logsip/logsip.go:39 +0x16f
main.main()
    /home/nick/go/logsip-demo/logsip-demo.go:8 +0x114
exit status 2
```

This is using the demo code as posted in the readme.go and when swapping out for the logsip.Default version:

```
package main

import "github.com/iamthemuffinman/logsip"

var log = logsip.Default()

func main() {
    log.Info("Just some info for ya")
    log.Warn("Some info you might need to know")
    log.Fatal("You done did something wrong")
    log.Panic("You REALLY done did something wrong")

    lulz := "lulz"

    log.Infof("Just some %s for ya", lulz)
    log.Warnf("You might need to know this %s", lulz)
    log.Fatalf("You done did the %s", lulz)
    log.Panicf("You REALLY done did the %s", lulz)

    log.Infoln("Just some info for ya")
    log.Warnln("Some info you might need to know")
    log.Fatalln("You done did something wrong")
    log.Panicln("You REALLY done did something wrong")
}

```

This is for go version go1.5.3 linux/amd64

**Comment by 17twenty** [comment]
I opened a pull request with a suggested fix.

**Comment by rdeusser** [comment]
Fixed with #4

**Comment by rdeusser** [comment]
Thanks for the PR!
